src/star-ratings.js

```diff
@@ -78,9 +79,7 @@ class StarRatings extends React.Component {
 
   get fillId() {
     // Using the rating's decimal place to set a unique id for the fill. e.g. a rating of 3.5 and 4.5 will use the same half fill star with id #star-grad-5.
-    return `star-grad-${Math.round((this.props.rating % 1) * 10)}${
-      this.props.name ? `-${this.props.name}` : ""
```

Comment:
if this `name` prop isnt being used anywhere else, I would remove it from the typings.

Reply:
Good call

Reply:
Leaving in, it's used